packages/types/v1/responses.ts

```diff
@@ -88,3 +89,17 @@ export const ZResponseUpdateInput = z.object({
 });
 
 export type TResponseUpdateInput = z.infer<typeof ZResponseUpdateInput>;
+
+export const ZResponseWithSurveyData = z.object({
```

Comment:
I think it would be better to have a type for the response card. We can maybe wait with this until we have the redesigned response card (PR in the making by contributor).

Reply:
yeah, we can wait on this prolly haha 🤞🏼